src/jobs/BasicMCJob.jl

```diff
@@ -114,18 +114,18 @@ BasicMCJob(
 ) =
   BasicMCJob(model, pindex, sampler, tuner, range, v0, outopts, resetpstate, verbose, check)
 
-function BasicMCJob{S<:VariableState}(
+function BasicMCJob(
   model::GenericModel,
-  sampler::MCSampler,
-  range::BasicMCRange,
-  v0::Dict{Symbol, S};
-  pindex::Integer=findfirst(v::Variable -> isa(v, Parameter), model.vertices),
-  tuner::MCTuner=VanillaMCTuner(),
-  outopts::Dict=Dict(:destination=>:nstate, :monitor=>[:value], :diagnostics=>Symbol[]),
-  resetpstate::Bool=true,
-  verbose::Bool=false,
-  check::Bool=false
```

Comment:
There are a number of other places in here where the bot did incorrect unindenting. I only reported it once to avoid spamming Keno with issues though. These should be fixed as well.